Server/tests/integration/test_manage_gameobject_look_at.py

```
@@ -0,0 +1,77 @@
+import pytest
+
+from .test_helpers import DummyContext
+import services.tools.manage_gameobject as manage_go_mod
+
+
+@pytest.mark.asyncio
+async def test_look_at_vector_target(monkeypatch):
+    """look_at action forwards look_at_target as a vector."""
+    captured = {}
+
+    async def fake_send(cmd, params, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `cmd`

(ARG001)